assets/src/tracking/trackError.js

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import { config } from './shared';
+import isTrackingEnabled from './isTrackingEnabled';
+import track from './track';
+
+/**
+ * @typedef {import('react').ErrorInfo} ErrorInfo
+ * Send an Analytics tracking error.
+ *
+ * @see https://developers.google.com/analytics/devguides/collection/ga4/exceptions
+ *
+ * @param {Error} error The error message.
+ * @param {ErrorInfo} [errorInfo] The React info object.
+ * @param {boolean} [fatal] Report whether there is a fatal error.
+ * @param {Object<*>} [additionalData] Additional event data to send.
+ * @return {Promise<void>} Promise that always resolves.
+ */
+//eslint-disable-next-line require-await
+async function trackError(
+  error,
+  errorInfo = null,
+  fatal = false,
+  additionalData = {}
+) {
+  if (!isTrackingEnabled()) {
+    return Promise.resolve();
+  }
+  const description = errorInfo
+    ? `${error.toString()}\n\n${errorInfo.componentName}\n${
+        errorInfo.componentStack
+      }`
+    : error.toString();
+  const eventData = {
+    send_to: config.trackingId,
+    event_category: 'error',
```

Comment:
That doesn't make sense I think? We already know it's an error because we track it as an exception below.

The event category should be something like "editor" or "dashboard" and be passed as an rgument to the function.

Reply:
Okay, thanks for clarifying, I think in the previous comment, you suggested only keep `error` and `errorInfo` from the passing arguments.